src/Symfony/Component/Cache/Adapter/TagAwareAdapter.php

```diff
@@ -292,12 +311,12 @@ public function reset(): void
 
     public function __sleep(): array
     {
-        throw new \BadMethodCallException('Cannot serialize '.__CLASS__);
+        throw new BadMethodCallException('Cannot serialize '.__CLASS__);
```

Comment:
this change looks wrong to me. I don't think we want those exception preventing serialization (for security reasons) to be caught when catching `Psr\Cache\CacheException`

Reply:
makes sense, reverted